### ISSUE TITLE
Aware of the History: Trajectory Forecasting with the Local Behavior Data

### ISSUE THREAD

**Issue by mpourkeshavarz** [Kay1794/Aware-of-the-history#1]
are you going to publish the code soon?

**Comment by Kay1794** [comment]
Yes, so sorry for the delay. The webpage will be updated once I finished cleaning up the code and the data.

**Comment by mpourkeshavarz** [comment]
Thanks for your prompt reply,
I'm interested in your work. Could you let me know how long it will take?

**Comment by Kay1794** [comment]
Hi, we just released the implementation of LaneGCN and have been preparing the P2T implementation. Please let me know if you see any issues. Thank you so much for your patience!

> Thanks for your prompt reply, I'm interested in your work. Could you let me know how long it will take?
